metricbeat/module/windows/service/service_status.go

```diff
@@ -234,11 +235,13 @@ func getServiceInformation(rawService *EnumServiceStatusProcess, servicesBuffer
 	if ServiceState(rawService.ServiceStatusProcess.DwCurrentState) != ServiceStopped {
 		processUpTime, err := getServiceUptime(rawService.ServiceStatusProcess.DwProcessId)
 		if err != nil {
+			if !errors.Is(err, os.ErrPermission) {
+				// if we have faced any other error, pass it to the caller
+				return service, err
+			}
 			if _, ok := protectedServices[service.ServiceName]; errors.Is(err, os.ErrPermission) && !ok {
```

Comment:
```suggestion
			if _, ok := protectedServices[service.ServiceName]; !ok {
```

this check seems redundant after https://github.com/elastic/beats/pull/43665/files#diff-9b90144972ecb21c1d6269ce8ae6dbe9a5b82f085fe2327d1a268f9f7d594bf5R238